benchmark/how-to-run-tpcc.md

```diff
@@ -137,15 +139,94 @@ loadWorkers=32  # The number of concurrent workers that load data.
+
+1. Modify the BenchmarkSQL configuration file.
+
+    The csv file of one warehouses requires 77 MB of disk space. To ensure sufficient disk space, add a line to the `benchmarksql/run/props.mysql` file:
+
+    ```text
+    fileLocation=/home/user/csv/  # Absolute path to directory where the csv files are stored
+    ```
+
+    The csv file name should adhere to the Lightning naming rules, namely `{database}.{table}.csv`, because you use TiDB Lightning to load data. Here you can modify the above configuration to:
+
+    ```text
+    fileLocation=/home/user/csv/tpcc.  # Absolute path to directory where the csv files are stored + file name prefix (database)
+    ```
+
+    In this case, the format of the csv file name is similar to `tpcc.bmsql_warehouse.csv`.
+
+2. Generate the csv file.
 
     {{< copyable "shell-regular" >}}
 
     ```shell
     ./runLoader.sh props.mysql
     ```
 
-This process might last for several hours depending on the machine configuration.
+3. Use Lightning to load data.
+
+    To load data using Lightning, see [TiDB Lightning Deployment](/reference/tools/tidb-lightning/deployment.md). This section will show you how to deploy Lightning using tidb-ansible to load data.
+
+    1. Edit `inventory.ini`.
+
+        It is recommended to specify the deployed IP, the port and the directory manually to avoid anomalies caused by conflicts. For the disk space of `import_dir`, see [TiDB Lightning Deployment](/reference/tools/tidb-lightning/deployment.md). `data_source_dir` refers to the directory where the csv files are stored, as mentioned above.
+
+        ```ini
+        [importer_server]
+        IS1 ansible_host=172.16.5.34 deploy_dir=/data2/is1 tikv_importer_port=13323 import_dir=/data2/import
+        [lightning_server]
+        LS1 ansible_host=172.16.5.34 deploy_dir=/data2/ls1 tidb_lightning_pprof_port=23323 data_source_dir=/home/user/csv
+        ```
+
+    2. Edit `conf/tidb-lightning.yml`.
+
+        ```yaml
+        mydumper:
+            no-schema: true
+            csv:
+                separator: ','
+                delimiter: ''
+                header: false
+                not-null: false
+                'null': 'NULL'
+                backslash-escape: true
+                trim-last-separator: false
+        ```
+
+    3. Deploy Lightning and Importer.
+
+        {{< copyable "shell-regular" >}}
+
+        ```shell
+        ansible-playbook deploy.yml --tags=lightning
+        ```
+
+    4. Start importing data.
+
+       * Log into the server where Lightning and Importer are deployed.
+       * Move into the deployment directory.
+       * Execute `scripts/start_importer.sh` under the Importer directory to start Importer.
+       * Execute `scripts/start_lightning.sh` under the Lightning directory to load data.
+
+       You can see the progress of the import process, or check whether the import process is completed through the log, because ansible is ued for deployment.
```

Comment:
```suggestion
       Because you've used TiDB Ansible deployment method, you can see the loading progress of Lightning on the monitoring page, or check whether the loading process is completed through the log.
```